about.html

```diff
@@ -0,0 +1,163 @@
+          galleries, libraries, museums, archives, governments and foundations,
+          lawyers, and activists - we hope you'll consider joining us this year
+          to build a stronger, more vibrant commons together.
+        </p>
+        <div class="contact">
+          Please contact via our email if you have any further questions about
+          this organization
+        </div>
+        <a class="email" href="mailto: email@domain.com">email@domain.com</a>
+      </section>
+    </div>
+
+    <!-- Section 2 -->
+    <section class="section-2">
+        <div class="text-container">
+          <h2 class="h2">
+            Creative Common<br class="br" />Global Summit 2023 Logo
+          </h2>
+          <hr class="red-line" />
+          <p class="mobile-p">
+            The Logo of Tech Global summit 2023 was decided through the logo
+            competition from 8. June to 7. July.
+          </p>
+  
+          <p class="desktop-p">
+            The logo of Tech Global Summit 2023 was decided through the logo
+            competition from 8. June to 7. July.
+            <br />
+            After online public voting and Selection Committee's judging process,
+            the logo from Naresh Agrawal won the prize. After consultation with
+            Mr. Agrawal we optimized the prize-winning-logo for this year's summit
+            as following:
+          </p>
+        </div>
+  
+        <img class="logo" src="images/Logopit_1684831391238.png" alt="image" />
+        <hr class="grey-line" />
+  
+        <div class="text-container">
+          <h2 class="h2">See the past Global summits</h2>
+          <hr class="red-line" />
+          <p class="desktop-p">
+            Take a look at the last two Global summits which took place in
+            Buenos Aires and Warsaw.
+          </p>
+        </div>
+  
+        <div class="event-container">
+          <div class="event event-1">
+            <h3 class="h3">2013</h3>
+            <p class="p">CC Global Summit 2013 in Buenos Aires</p>
+          </div>
+          <div class="event event-2">
+            <h3 class="h3">2011</h3>
+            <p class="p">CC Global Summit 2011 in Warsaw</p>
+          </div>
+        </div>
+      </section>
+
+      <!-- Footer Section -->
+    <footer class="footer">
+        <img class="logo" src="images/Logopit_dark.png" alt="image" />
+        <div class="text">
+          <div class="text-inner-container">
+            <div class="creative">2023 Global Tech India.</div>
+            <div class="some">Some Rights Reserved.</div>
+          </div>
+  
+          <div class="licence">
+            Unless otherwise specified, all content on the website is licensed under a Global Tech Attribution 2.0 India license.
+            See http://globaltech.org/licences/by/2.0/in for detailed usage features and permission conditions.
+          </div>
+        </div>
+      </footer>
```

Comment:
- [ ] You seem to be missing the partner's section on the About page of your mobile version here, as you can see in the original design the partner's section is only hidden in the desktop version and it's visible in the mobile version. 😊
- Your design:
  
![image](https://github.com/suthish777/capstone__01/assets/104968236/c9b2eb9a-ea11-4098-abdb-e4f688cfb457)
- Original design:
  ![image](https://github.com/suthish777/capstone__01/assets/104968236/b3ebc605-6202-4e53-a1d5-e791e08ef53b)